fc/parsing/CompactSyntaxParser.py

```diff
@@ -996,7 +1015,7 @@ def _expr(self):
             args.extend([output_names, optional_flags])
             model = Model.NestedProtocol(*args)
             result = Simulations.OneStep(0)
-            result.SetModel(model)
+            result.set_model(model)
```

Comment:
I'd rather not do this kind of renaming piecemeal, as it leads to an inconsistent API. Rather we should do it in one (huge!) PR for the entire project.

Reply:
OK, will revert!